packages/blade/src/components/Button/Button/Button.tsx

```diff
@@ -104,13 +133,17 @@ const _Button: React.ForwardRefRenderFunction<BladeElementRef, ButtonProps> = (
     <BaseButton
       {...(icon ? { icon, children } : { children })}
       {...getStyledProps(rest)}
+      id={id}
       ref={ref}
       href={href}
       target={target}
       rel={rel}
       accessibilityProps={{
         label: accessibilityLabel,
         describedBy: rest['aria-describedby'],
+        expanded: rest['aria-expanded'] === 'true',
```

Comment:
This will condition will not work if consumer passes the prop like an actual boolean 

```
<Button aria-expanded />
<Button aria-expanded={true} />
```

Since the type of aria props are `Booleanish`

```ts
type Booleanish = boolean | "true" | "false";
```

Reply:
I think you can just pass it has is without checking for `=== true` part that should work.

Reply:
oh yes I changed this in the other stacked PR so its resolved